src/main/java/nexus/task/Event.java

```diff
@@ -0,0 +1,42 @@
+package nexus.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Event class that inherits from Task.
+ */
+public class Event extends Task{
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+
+    /**
+     * Create an event using description, start date & time and end date & time.
+     *
+     * @param description String
+     * @param from Start date & time
+     * @param to End Date & time
```

Comment:
I believe there should be punctuation at the end of every parameter description.